internal/pkg/saconfig/config.go

```diff
@@ -21,11 +21,11 @@ type EventConfiguration struct {
 	UniqueName          string         `json:"UniqueName"`
 	ServiceType         string         `json:"ServiceType"`
 	IgnoreString        string         `json:"-"` //TODO(mmagr): ?
-	UseTls              bool           `json:"UseTls"`
-	TlsServerName       string         `json:"TlsServerName"`
-	TlsClientCert       string         `json:"TlsClientCert"`
-	TlsClientKey        string         `json:"TlsClientKey"`
-	TlsCaCert           string         `json:"TlsCaCert"`
+	UseTLS              bool           `json:"UseTls"`
+	TLSServerName       string         `json:"TlsServerName"`
```

Comment:
So APIContext is not ok, but TLSServer... is ok? Do we want to have consistent naming rules? Or is it the lint being that much inconsistent?

Reply:
Changing `APIContext` is because it's unnecessary to have `API` in the name when calling it as the `api` package. You end up calling via `api.APIContext` vs `api.Context`. The extra `API` component is superfluous because the package is already `api`.

It's not a case/camelcase linting issue but rather an overloaded naming issue.

Reply:
Ok, fair enough. I wasn't taking the namespace into the account.

Reply:
I wouldn't have even noticed really if the linting didn't specifically call it out and say it :)